### PR TITLE
MemoEdit: functionality with SCSS

### PR DESCRIPTION
**Pull request by steal4life**
## Description

Closes #insert_issue_number

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)

## Checklist:

- [ ] The PR is submitted to the correct branch (`master` for dev, `rel-1.x` for maintenance).
- [ ] My code follows the code style of this project.
- [ ] I've added relevant tests.

### CODE REVIEW

**Comment by steal4life** [comment]
closes #5026

WIP!

File: Source/Blazorise/Components/MemoEdit/MemoEdit.razor.cs [hunk 1]
```diff
@@ -2,66 +2,22 @@
...
-using Microsoft.AspNetCore.Components.Web;
 #endregion
 
 namespace Blazorise;
 
 /// <summary>
 /// Component that allows you to display and edit multi-line text.
 /// </summary>
-public partial class MemoEdit : BaseInputComponent<string>, ISelectableComponent, IAsyncDisposable
+public partial class MemoEdit : BaseInputComponent<string>, ISelectableComponent
```
Comment (stsrki) [inline Source/Blazorise/Components/MemoEdit/MemoEdit.razor.cs:14]: Don't remove `IAsyncDisposable`.

File: Source/Blazorise/Components/MemoEdit/MemoEdit.razor [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
+    protected override async Task OnFirstAfterRenderAsync()
+    {
+        AutoSize();
+        await base.OnFirstAfterRenderAsync();
+    }
+
+    private void AutoSize()
+    {
+        var textarea = ElementRef;
+        textarea.Style.Height = "auto";
```
Comment (stsrki) [inline Source/Blazorise/Components/MemoEdit/MemoEdit.razor:20]: Styles should not be defined like this. You need to override `BuildStyles`

```cs
protected override void BuildStyles( StyleBuilder builder )
{
    // do it here

    base.BuildStyles( builder );
}
```

Also, the new styles must be used only when needed.

File: Source/Blazorise/Components/MemoEdit/MemoEdit.razor.cs [hunk 3]
```diff
@@ -159,64 +91,23 @@ protected Task OnChangeHandler( ChangeEventArgs e )
     }
 
     /// <inheritdoc/>
-    protected async Task OnInputHandler( ChangeEventArgs e )
+    protected Task OnInputHandler( ChangeEventArgs e )
```
Comment (stsrki) [inline Source/Blazorise/Components/MemoEdit/MemoEdit.razor.cs:94]: Could you let me know why you removed Debounce logic? It must be kept.

File: Source/Blazorise/Components/MemoEdit/MemoEdit.razor.cs [hunk 4]
```diff
@@ -159,64 +91,23 @@ protected Task OnChangeHandler( ChangeEventArgs e )
...
-    {
-        InvokeAsync( () => CurrentValueHandler( value ) );
+        return Task.CompletedTask;
     }
 
     /// <inheritdoc/>
     public virtual async Task Select( bool focus = true )
     {
-        await JSUtilitiesModule.Select( ElementRef, ElementId, focus );
+        // Ovo je mjesto gdje biste inače koristili JS za selekciju.
```
Comment (stsrki) [inline Source/Blazorise/Components/MemoEdit/MemoEdit.razor.cs:107]: Don't remove existing logic.

**Comment by stsrki** [comment]
Closing since, after further investigation, it seems that relying just on CSS for this feature would work only partially. And some features like TabSize and SoftTabs would stop working. Also, the use of a wrapper would break a lot of CSS for forms.

**Pull request closed without merging**